tests/animal_module_tests/test_animal_life_cycle.py

```diff
@@ -1463,3 +1466,171 @@ def test_reset_daily_stats(life_cycle_manager: LifeCycleManager) -> None:
+                                  net_energy_diet_concentration=(15.625 * 0.64)/10.0,
+                                  days_born=heiferI.days_born)
+
+
+def test_heiferI_calc_manure_excretion(mocker: MockerFixture) -> None:
+    """Unit test for the function calc_manure_excretion in file routines/animal/life_cycle/heiferI.py."""
+
+    heiferI = mocker.MagicMock(autospec=HeiferI)
+    mocker.patch('RUFAS.routines.animal.life_cycle.heiferI.HeiferI.__init__', return_value=heiferI)
+    heiferI.body_weight = 1000.0
+    heiferI.mature_body_weight = 1200.0
+    heiferI.daily_growth = 1.0
+    heiferI.days_born = 300
+    feed = mocker.MagicMock(autospec=Feed)
+    mock_numI = (mocker.MagicMock(autospec=int), mocker.MagicMock(autospec=int))
+    heiferI.calc_base_manure.return_value = mock_numI
+    HeiferI.calc_manure_excretion(heiferI, feed, "methane_model")
+    heiferI.calc_base_manure.assert_called()
+
+
+def test_heiferI_phosphorus_rqmts(mocker: MockerFixture) -> None:
+    """Unit test for the function phosphorus_rqmts in file routines/animal/life_cycle/heiferI.py."""
+
+    heiferI = mocker.MagicMock(autospec=HeiferI)
+    mocker.patch('RUFAS.routines.animal.life_cycle.heiferI.HeiferI.__init__', return_value=heiferI)
+    heiferI.p_maint_feces = 10
+    heiferI.p_req = 0
+    HeiferI.phosphorus_rqmts(heiferI, 10)
+    p_urine = 0.000002 * heiferI.body_weight * 1000
+
+    heiferI.p_growth = (
+        (0.0012 + 0.004635 * (heiferI.mature_body_weight**0.22) * (heiferI.body_weight ** (-0.22)))
+        * heiferI.daily_growth
+        / 0.96
+        * 1000
+    )
+    p_absorb = p_urine + heiferI.p_maint_feces + heiferI.p_growth
+    p_req = p_absorb / 0.664
+    assert heiferI.p_maint_feces == 8
+    assert heiferI.p_req == p_req
+
+
+def test_heiferI_get_non_preg_bw_change(mocker: MockerFixture) -> None:
+    """Unit test for the function get_non_preg_bw_change in file routines/animal/life_cycle/heiferI.py."""
+
+    heiferI = mocker.MagicMock(autospec=HeiferI)
+    heiferI.body_weight = 1000.0
+    heiferI.mature_body_weight = 1200.0
+    heiferI.daily_growth = 1.0
+    heiferI.days_born = 300
+    heifer_repro_method = 'TAI'
+    heifer_TAI_protocol = 'd5CG2P'
+    heifer_synchedED_protocol = '2P'
+    nutrient_standard = 'NRC'
+    animal_base_config = {
+        "heifer_repro_method": heifer_repro_method,
+        "target_heifer_preg_day": 300,
+        "breeding_start_day_h": 400,
+        "heifer_repro_programs": {
+            "heifer_TAI_protocol": heifer_TAI_protocol,
+            "heifer_synchED_protocol": heifer_synchedED_protocol
+        },
+        "nutrient_standard": nutrient_standard
+    }
+    mocker.patch('RUFAS.routines.animal.life_cycle.life_cycle.AnimalBase.config', animal_base_config)
+    weight = HeiferI.get_non_preg_bw_change(heiferI)
+    calc_weight = 0.55 * 0.96 * heiferI.mature_body_weight - 0.96 * heiferI.body_weight  # divisor == 1
+    assert weight == calc_weight
```

Comment:
```suggestion
    mocker.patch('RUFAS.routines.animal.life_cycle.life_cycle.AnimalBase.config', animal_base_config)
    
    actual_weight = HeiferI.get_non_preg_bw_change(heiferI)
    
    expected_weight = 0.55 * 0.96 * heiferI.mature_body_weight - 0.96 * heiferI.body_weight  # divisor == 1
    assert actual_weight == expected_weight
```
Grouping by setup, act, and assert helps to keep things more readable. Also, using the terms actual and expected for what the method actually does vs what you expect it to do also makes the test a lot more readable.